pandas/core/frame.py

```diff
@@ -6037,7 +6037,7 @@ def _gotitem(self,
         # TODO: _shallow_copy(subset)?
         return subset[key]
 
-    _agg_doc = dedent("""
+    _agg_see_also_doc = dedent("""
```

Comment:
sorry, just realized, this is not only the see also doc, can you update the name?

Reply:
Where do you mean, I only see one occurrence in frame.py?

Reply:
The name of this variable is `_agg_see_also_doc`, but in the content, there is the summary and the see also, so the name is misleading. `_agg_summary_and_see_also_doc` would be more appropriate (or something else)

Reply:
Ah I understand, thanks for clarifying :)